### ISSUE TITLE
Does not work with 2022 Frame TV - LS03B - with 1602 Firmware - Art Mode API apparently removed by Samsung

### ISSUE THREAD

**Issue by Spinner65** [ow/samsung-frame-art#8]
as of January 13, 2023 it DOES NOT WORK with 2022 Frame TV - LS03B-  running 1602 Firmware

-running the script in debug mode doesn't show a connect error.
-accessing the http://<your-ip>:8001/api/v2/ works
-you can use the xchwarze api code to do things like power the TV off and on.

BUT :
"Samsung decided to remove the API for Art Mode from newer TVs. Unfortunately they didn't offer an alternative yet. Only TVs from 2022 or newer are affected, TVs running Tizen 6.5 OS or newer"
https://tavicu.github.io/homebridge-samsung-tizen/configuration/frame-tvs.html

https://github.com/xchwarze/samsung-tv-ws-api/issues/108

**Comment by NickWaterton** [comment]
As of firmware 1622, the art WebSocket api is back!

With some changes, but mostly works as it used to.

**Comment by mjjjjjjlindow** [comment]
I am trying to get this working on a 2022 Frame TV with 1622 firmware. What are the changes you made to get it working?

**Comment by NickWaterton** [comment]
Significant re-write.

See my fork at https://github.com/NickWaterton/samsung-tv-ws-api

**Comment by atoulmin** [comment]
I have an 2020 frame tv and this works, however my new model does not work with this.  Any ideas on getting this going?

**Comment by NickWaterton** [comment]
Are you using my fork, specifically for the Frame TV?

https://github.com/NickWaterton/samsung-tv-ws-api

**Comment by atoulmin** [comment]
> Are you using my fork, specifically for the Frame TV?
> 
> https://github.com/NickWaterton/samsung-tv-ws-api

Yeah i am, it will work if i activate art mode first on the tv by remote then it will send the artwork.  Otherwise it won't.

**Comment by fishfisher** [comment]
FYI, this is suddenly working again for 2022 The Frame😊

**Comment by NickWaterton** [comment]
Sorry, what is working?
